app/views/item_seasons/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<p id="notice"><%= notice %></p>
+
+<h1 class='page__title'> <%= formatted_month_name(@item_season.month_index).capitalize %> </h1>
+
+<table class='produce-table'>
+  <% ItemSeason.includes([:produce_item]).where(month_index: @item_season.month_index).find_each do |season| %>
```

Comment:
`ItemSeason.includes([:produce_item]).where(month_index: @item_season.month_index)` should probably be assigned in the controller

Reply:
I don't think the name communicates what's stored in the instance variable. In the end, you are still referring to a sequence of item seasons. The fact that you later find the associated produce shouldn't matter in the naming. Does that make sense?

Reply:
I still think you are not fetching produce, but seasons. If you care about produce item, you should invert the query:

`ProduceItem.joins(:item_seasons).where(item_seasons: { month_index: @item_season.month_index })` and then assign to something like `@produce_items` or `@season_produce_items`

I think I misanderstood in the first comment, I should have suggested this from the get-go.